### ISSUE TITLE
Bug in docker file

### ISSUE THREAD

**Issue by vsrivatsa1** [quic/gunyah-support-scripts#6]
I think dockerfile-hyp needs to have this fix:

- COPY ./utils/* "${BASE_DIR}/utils/"
+ COPY --chown=$USER:$USER ./utils/* "${BASE_DIR}/utils/"

Otherwise some of the cp commands fail with perm denied:

    cp ${BASE_DIR}/utils/kern-test.sh ${TOOLS_DIR}/qemu/  && \

**Comment by quic-yvasi** [comment]
Thanks Srivatsa, I didn't see issue on internal Linux machines, I will check it out on external machine and will incorporate the fixes in next release.

Thanks
Yug

**Comment by quic-yvasi** [comment]
Hi Srivatsa, thanks for raising this, I found that I found similar issue on one of my home machines. 

I was seeing issue of the ownership and mode not changing despite an explicit command to do so. Seems like this is issue with Ubuntu provided docker.io installation (https://github.com/moby/moby/issues/46161).

So, your change should fix the problem you are facing and any further issues of permission denials are causing by wrong variant of docker installation.

Thanks
Yug
